docs/_documentations/mdt-che-installinfo.md

```diff
@@ -139,12 +139,12 @@ Go to **View**>**Find Command…**>**Codewind: Build**.
 ## Updating the version
 Restart the Codewind workspace in Che. Che automatically pulls the newest version of Codewind and the Theia extension.
 
-# Installing Codewind on OpenShift
+## Installing Codewind on OpenShift
 
-## Codewind OpenShift Do (odo) extension 
-See the [`codewind-odo-extension` repository](https://github.com/eclipse/codewind-odo-extension) for the extension to Codewind that provides support for OpenShift projects.
+### Codewind OpenShift Do (odo) extension 
```

Comment:
The purpose of this file has been updated to only focus on install information, so we can remove this section `Codewind OpenShift Do (odo) extension` since it's feature overview related information. I talk with Sarah and Jake before, I can work on creating a new file and move all ODO extension related information there.

Reply:
Ok, great, removed. I did consider putting all the `Codewind OpenShift Do (odo) extension` content in a new file but then decided to focus on the specific remarks in the issue - we can iterate.